uncertainties/core.py

```diff
@@ -1044,3 +1046,34 @@ def ufloat(nominal_value, std_dev=None, tag=None):
     """
 
     return Variable(nominal_value, std_dev, tag=tag)
+
+
+# Deprecated UFloat methods
+
+
+def deprecation_wrapper(func, msg):
+    @functools.wraps(func)
+    def wrapped(*args, **kwargs):
+        warn(msg, FutureWarning)
```

Comment:
I think you want stackevel 2 so the warning doesn't blame the wrapper function.

```suggestion
        warn(msg, FutureWarning, stacklevel=2)
```